man/lcovrc.5

```diff
@@ -281,6 +288,14 @@ lcov_function_coverage = 1
 .br
 lcov_branch_coverage = 0
 .br
+
+# ask LCOV to return non-zero exit code if line coverage is
```

Comment:
Please start the description with a capital letter to be consistent with the remainder of the man page example.